### ISSUE TITLE
Can you share the model file that your Single-task Model trained on the verse2020 dataset, I can't achieve the effect in the paper?

### ISSUE THREAD

**Issue by XFivezzz** [yeerwen/UniSeg#15]
![1693564195521](https://github.com/yeerwen/UniSeg/assets/93257077/6b9a868f-e036-4120-93c6-372c038a5039)

**Comment by yeerwen** [comment]
Did you add a deep supervision strategy to these single-task models and implement them based on the nnUNet framework?

**Comment by XFivezzz** [comment]
Thanks for your reply，Yes, I did，Have you measured the IoU, HD, Percision, and Recall metrics of these single-task models?I asked for the model trained in the experimental environment of the paper in order to better validate these indicators.

**Comment by yeerwen** [comment]
No, we did not measure other metrics. For the competitive approach, our modifications are (1) training the model based on the nnUNet framework; and (2) adding a deep supervision strategy to each decoder output. We report the key hyperparameters, such as learning rate and optimizer, in the appendix material.
